protocol/development-workflow/README.md

```diff
@@ -162,6 +168,8 @@ their release branch onto a staging server.
 
 > Don't forget to update the [Server Status Spreadsheet](https://docs.google.com/spreadsheets/d/1qZ5x80cYXHxACJbZ20W5MqH6ETRFDmaLTnsjaqen6O0/edit#gid=0) before and after you perform QA.
 
+In the case of a gem update, this protocol should be followed. QA is done by deploying to a staging server and ensuring that the application doens't crash.
```

Comment:
I would argue adding this line to the end of the section, like:

> **Note**: In the case of a dependency update, QA involves deploying to a staging server and ensuring that the application doesn't crash.

I also like using `dependency` here instead of `gem`, in case we update Ruby, or down the line use Webpacker for NPM packages.